### PR TITLE
updating recipes for pcaExplorer and ideal

### PR DESCRIPTION
**Pull request by federicomarini**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bioconductor-ideal/meta.yaml [hunk 1]
```diff
@@ -10,28 +10,28 @@ source:
...
-    - 'bioconductor-goseq >=1.32.0,<1.34.0'
-    - 'bioconductor-gostats >=2.46.0,<2.48.0'
-    - 'bioconductor-ihw >=1.8.0,<1.10.0'
-    - 'bioconductor-iranges >=2.14.12,<2.16.0'
-    - 'bioconductor-limma >=3.36.5,<3.38.0'
-    - 'bioconductor-pcaexplorer >=2.6.0,<2.8.0'
-    - 'bioconductor-s4vectors >=0.18.3,<0.20.0'
-    - 'bioconductor-summarizedexperiment >=1.10.1,<1.12.0'
-    - 'bioconductor-topgo >=2.32.0,<2.34.0'
+    - bioconductor-annotationdbi
```
Comment (bgruening) [inline recipes/bioconductor-ideal/meta.yaml]: please keep the pinning and update it to the new release.

File: recipes/bioconductor-pcaexplorer/meta.yaml [hunk 2]
```diff
@@ -10,26 +10,26 @@ source:
...
-    - 'bioconductor-genefilter >=1.62.0,<1.64.0'
-    - 'bioconductor-genomicranges >=1.32.7,<1.34.0'
-    - 'bioconductor-go.db >=3.6.0,<3.8.0'
-    - 'bioconductor-gostats >=2.46.0,<2.48.0'
-    - 'bioconductor-iranges >=2.14.12,<2.16.0'
-    - 'bioconductor-limma >=3.36.5,<3.38.0'
-    - 'bioconductor-s4vectors >=0.18.3,<0.20.0'
-    - 'bioconductor-summarizedexperiment >=1.10.1,<1.12.0'
-    - 'bioconductor-topgo >=2.32.0,<2.34.0'
+    - bioconductor-annotationdbi
```
Comment (bgruening) [inline recipes/bioconductor-pcaexplorer/meta.yaml]: same here use pinning for the new bioc release.

**Comment by federicomarini** [comment]
Pinnings have been updated ;)

**Comment by dpryan79** [comment]
No longer needed

**Pull request closed without merging**